### PR TITLE
Chores/adapt schedule

### PR DESCRIPTION
**Pull request by chrysalag**
# [Story in Trello](https://trello.com/c/inj1vd5w/25-1-adapt-the-new-schedule-from-volunteer-also-to-klient-profile)

### Done for approval?

* [x] Acceptance criteria are met
* [x] Code quality checks are green
* ~[ ] Readme updated if needed~
* [x] Story under test
* [x] Mobile works
* [x] Seeds created
* [x] Translated
* [x] Code is reviewed

### Done after the merge?

* [ ] Deployed to staging after the merge
* [ ] Tested manually on staging

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 module ApplicationHelper
+  def availability_collection
+    [:flexible, :morning, :afternoon, :evening, :workday, :weekend]
```
Comment (nuritnt) [inline app/helpers/application_helper.rb]: if we have this as a concern, why do we need it here again? for the translations?

File: app/models/volunteer.rb [hunk 2]
```diff
@@ -82,11 +82,6 @@ def self.state_collection
                           :dancing, :health, :cooking, :excursions, :women, :teenagers,
                           :children, :other_offer].freeze
   REJECTIONS = [:us, :her, :other].freeze
-  AVAILABILITY = [:flexible, :morning, :afternoon, :evening, :workday, :weekend].freeze
-
-  def self.human_boolean(boolean)
-    boolean ? I18n.t('simple_form.yes') : I18n.t('simple_form.no')
```
Comment (nuritnt) [inline app/models/volunteer.rb]: why did you remove this? 
I thought this was used for example in the checklist which puts `false = no`, `true = yes`

Comment (chrysalag) [inline app/models/volunteer.rb]: We had this function twice.

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (chrysalag) [inline app/helpers/application_helper.rb]: The concern in a controller concern, used by the controllers and this is a helper method to be used by the views. I am not sure if concerns are supposed to be used somehow by the views.

Comment (nuritnt) [inline app/helpers/application_helper.rb]: ah yes totally true :D
